<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoostSettingsPage</class>
 <widget class="QWidget" name="BoostSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Log format:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="logFormatCB"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Report level:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="reportLevelCB"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="randomizeCB">
           <property name="toolTip">
            <string>Randomize execution order.</string>
           </property>
           <property name="text">
            <string>Randomize</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Seed:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="seedSB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>A seed of 0 means no randomization. A value of 1 uses the current time any other value is used as random seed generator.</string>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="systemErrorCB">
         <property name="toolTip">
          <string>Catch or ignore system errors.</string>
         </property>
         <property name="text">
          <string>Catch system errors</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fpExceptions">
         <property name="toolTip">
          <string>Enable floating point exception traps.</string>
         </property>
         <property name="text">
          <string>Floating point exceptions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="memoryLeakCB">
         <property name="toolTip">
          <string>Enable memory leak detection.</string>
         </property>
         <property name="text">
          <string>Detect memory leaks</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>84</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
